File: backend-model/src/main/java/outcobra/server/model/dto/MarkReportEntryDto.java
package outcobra.server.model.dto;

import outcobra.server.model.MarkReportEntry;
import outcobra.server.model.interfaces.MappableDto;
import outcobra.server.model.interfaces.Mapper;

public class MarkReportEntryDto implements MappableDto<MarkReportEntryDto, MarkReportEntry> {
    @Override
    public Mapper<MarkReportEntryDto, MarkReportEntry> getMapper() {
        throw new UnsupportedOperationException("not implemented");
    }

    @Override
    public MarkReportEntry toEntity() {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: backend/src/main/kotlin/outcobra/server/model/dto/emptyDto/MarkReportEntryDto.kt
package outcobra.server.model.dto.emptyDto

import outcobra.server.model.MarkReportEntry
import outcobra.server.model.mapper.MappableDto
import outcobra.server.model.mapper.Mapper

class MarkReportEntryDto : MappableDto<MarkReportEntryDto, MarkReportEntry> {
    override fun getMapper(): Mapper<MarkReportEntryDto, MarkReportEntry> {
        throw UnsupportedOperationException("not implemented")
    }

    override fun toEntity(): MarkReportEntry {
        throw UnsupportedOperationException("not implemented")
    }
//TODO Implement
}